src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/resource/VisitResource.kt

```diff
@@ -261,4 +262,64 @@ class VisitResource(private val visitService: VisitService) {
         ignoreMissingRoom = ignoreMissingRoom
       )
     )
+
+  @PreAuthorize("hasRole('ROLE_NOMIS_VISITS')")
+  @GetMapping("/visits/rooms/usage-count")
+  @ResponseStatus(HttpStatus.OK)
+  @Operation(
+    summary = "get visit room usage by filter",
+    description = "Retrieves a list of rooms with usage count for the (filtered) visits",
+    responses = [
+      ApiResponse(
+        responseCode = "200",
+        description = "Pageable list of visit ids is returned"
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized to access this endpoint",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class)
+          )
+        ]
+      ),
+    ]
+  )
+  fun getVisitRoomCountsByFilter(
+    @PageableDefault(sort = ["whenCreated"], direction = Sort.Direction.ASC)
+    pageRequest: Pageable,
+    @RequestParam(value = "prisonIds", required = false)
+    @Parameter(
+      description = "Filter results by prison ids (returns all prisons if not specified)",
+      example = "['MDI','LEI']"
```

Comment:
is this the correct syntax?  I thought `example` could take an array instead rather than a string with arrays inside?